bitsail-test/bitsail-test-integration/bitsail-test-integration-doris/src/test/java/com/bytedance/bitsail/test/integration/doris/source/DorisSourceITCase.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.test.integration.doris.source;
+
+import com.bytedance.bitsail.common.configuration.BitSailConfiguration;
+import com.bytedance.bitsail.connector.doris.option.DorisReaderOptions;
+import com.bytedance.bitsail.test.integration.AbstractIntegrationTest;
+import com.bytedance.bitsail.test.integration.utils.JobConfUtils;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore
+public class DorisSourceITCase extends AbstractIntegrationTest {
+
+  @Test
+  public void test() throws Exception {
+    BitSailConfiguration jobConf = JobConfUtils.fromClasspath("doris_to_print.json");
+    addDorisInfo(jobConf);
+    submitJob(jobConf);
+  }
+
+  /**
+   * Add your doris setting to job configuration.
+   * Below codes are just example.
+   * <p>
+   * select id, bigint_type, string_type, double_type from doris_table where id = 1
+   */
+  public void addDorisInfo(BitSailConfiguration jobConf) {
```

Comment:
@DongLiang-0 Thanks for your contribution! Are you using your local environment for testing? I am testing doris myself recently, and found we can actually use test container to simplify the testing experience. It took me a while to start a doris cluster in my local machine. If you are interested, we can actually add the doris test container with this PR. Maybe contribute to the testcontainer project as well. [DockerComposeContainer](https://www.testcontainers.org/modules/docker_compose/)

Reply:
Thanks for your suggestion, I will try